### ISSUE TITLE
Sort all translation items alphabetically

### ISSUE THREAD

**Issue by jmcameron** [IMA-WorldHealth/bhima#280]
Sort all translation items alphabetically to make it easier to find items and to simplify textual comparisons between versions.

NOTE: the utility 'utilities/translation/tfsort.js' will perform this sorting.   The script needs to be applied and double-checked to make sure no language terms are lost in the process.

**Comment by jniles** [comment]
This is a great idea.  Next week, @sfount and I will be pretty much offline and that would be the best time to work out these rather large changes.  It will certainly make future development easier.

**Comment by jmcameron** [comment]
I plan to do this tomorrow, test it, and push into into development myself.   It is a simple fix and one that I can test several ways to make sure that it is okay.  I found out that jsonlint will sort a file so I will double-check against that.   I've warned the team here that they should expect merge conflicts in the two language files for any outstanding pull requests that have mods on those files.  Once the merge is done, I'll send an email to all about the easiest way to deal with these merge conflicts.
